lib/pharos/phase_manager.rb

```diff
@@ -4,6 +4,13 @@ module Pharos
   class PhaseManager
     include Pharos::Logging
 
+    RETRY_ERRORS = [
+      OpenSSL::SSL::SSLError,
+      Kubeclient::HttpError,
+      Net::SSH::Disconnect,
```

Comment:
I doubt that this retry loop by itself would be enough to reconnect on `Net::SSH` errors.

Reply:
True, I'll remove `Net::SSH::*`.